src/server/api/controllers/messages-controller.js

```diff
@@ -78,15 +78,30 @@ const deleteMessage = async (messagesId) => {
 };
 
 const createMessage = async (body) => {
-  await knex('channel_messages').insert({
+  const newMessage = {
     message: body.message,
     created_at: moment().format(),
     fk_user_id: body.userId,
     fk_channel_id: body.channelId,
+  };
+  const messageId = await knex('channel_messages').insert(newMessage);
+  const usersInChannel = knex
+    .select('fk_user_id')
+    .from('channel_members')
+    .where({ fk_channel_id: body.channelId });
+  const newRowForEachUser = await usersInChannel;
```

Comment:
Remove this intermediate variable `newRowForEachUser ` and move `await` to line 88.